l10n/en-US/viewer.ftl

```diff
@@ -351,6 +351,8 @@ pdfjs-editor-stamp-add-image-button =
 pdfjs-editor-stamp-add-image-button-label = Add image
 # This refers to the thickness of the line used for free highlighting (not bound to text)
 pdfjs-editor-free-highlight-thickness-input = Thickness
+pdfjs-editor-free-highlight-thickness-title =
+    .title = Change thickness when highlighting things that aren't text
```

Comment:
Is this coming from Content? It sounds a bit funny to me. Maybe this, but I'm not a native speaker.

> Change thickness when highlighting elements other than text

Reply:
I understand the concern. How about `Change thickness when highlighting items other than text` ? This is from Content

Reply:
That works for me 👍🏼